src/incubator/Slider/index.tsx

```diff
@@ -278,10 +283,22 @@ const Slider = React.memo((props: Props) => {
     }
   }, 200), [onValueChange]);
 
+  const _onValueChange = useCallback((value: number) => {
```

Comment:
Why not pass the throttle wait (200/100 to 0) instead of having code duplications?

Reply:
added throttleTime prop instead 🙏